src/components/Table/GridTable.test.tsx

```diff
@@ -1136,7 +1136,12 @@ describe("GridTable", () => {
       },
     ];
     const api: MutableRefObject<GridTableApi<NestedRow> | undefined> = { current: undefined };
```

Comment:
Even those pages don't have to use the MutableRef anymore, these tests do so that they can export the api out of the test component and up into the test itself (the test can't do `const api = useGridTableApi()` b/c hooks aren't allowed to be instantiated outside of a component)